RecoParticleFlow/PFClusterProducer/plugins/LegacyPFClusterProducer.cc

```diff
@@ -68,7 +66,6 @@ class LegacyPFClusterProducer : public edm::stream::EDProducer<> {
     edm::ParameterSetDescription desc;
     desc.add<edm::InputTag>("src");
     desc.add<edm::InputTag>("PFRecHitsLabelIn");
-    desc.add<edm::ESInputTag>("pfClusterParams");
```

Comment:
relaying an observation from @missirol, if this module has to be parsed in confDB we need to provide defaults in the `fillDescription`

Reply:
Indeed, after the removal of ```pfClusterParams``` we still need default values in fillDescriptions added for:
```
InputTag src
InputTag PFRecHitsLabelIn
InputTag recHitsSource
```
Empty InputTags are fine, but the values must be there.